components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/pages/cdmf.page.groups/groups.hbs

```diff
@@ -226,8 +226,7 @@
                     <div class="content">
                         <div class="row">
                             <div class="col-lg-5 col-md-6 col-centered">
-                                <h3>Exception at backend. Try Later.</h3>
-
+                                <h3 id="error-msg">Bad Request. Please contact your administrator.</h3>
```

Comment:
If it's a bad-request, definitely it's a problem with the request from UI which can be considered as a programming error at the UI level, not adhering to the corresponding back-end APIs. So, this is infact a type of error message, that we cannot expect a UI to show.